### DISCUSSION TITLE
LEDs not turning on

### DISCUSSION THREAD

**Discussion by CZonin** [awawa-dev/HyperSerialESP32#60] [Announcements]
Just flashed `firmware_esp32_SK6812_RGBW_NEUTRAL.bin` on https://www.amazon.com/gp/product/B0CNVZPYFC/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1 which contains this board: 
![PXL_20240613_170655006](https://github.com/awawa-dev/HyperSerialESP32/assets/67965669/b96d2a44-7939-4d21-9eb5-dbcf119612f4)
I'm using it with an SK6812 Natural White.

It flashed successfully and HyperHDR is recognizing the device, but my LEDs are not turning on. I was previously using it with WLED and didn't have any issues. Any ideas?

**Comment by awawa-dev** [comment]
Hi
And are sure that the data GPIO on the output is correct for the standard HyperSerialESP32 firmware?

**Reply by CZonin** [reply]
Just saw the pinout section in the documentation, sorry I don't have too much experience with all of this. Just switched it over to GPIO2 and it seems to be working. Thanks for pointing that out!
